docs/docs/configuration/object_detectors.md

```diff
@@ -252,7 +254,7 @@ detectors:
     device: 0 #This is the default, select the first GPU
 
 model:
-  path: /config/model_cache/tensorrt/yolov7-tiny-416.trt
+  path: /config/model_cache/tensorrt/yolov7-320.trt
```

Comment:
If the default model here is going to be 320 then the width and height need to be 320 in the config as well